python/pyspark/ml/deepspeed/deepspeed_distributor.py

```diff
@@ -61,6 +61,30 @@ def __init__(
             The configuration file to be used for launching the deepspeed application.
             If it's a dictionary containing the parameters, then we will create the file.
             If None, deepspeed will fall back to default parameters.
+
+        Examples
+        --------
+        Run Deepspeed training function on a single node
+
+        >>> def train(learning_rate):
+                import deepspeed
+                # rest of training function
+                return model
+        >>> distributor = DeepspeedTorchDistributor(num_gpus=4,
```

Comment:
All these arguments should be like `numGpus` because PySpark follows camelCase for user facing API. Another option is to make the argument one word. like `ngpus`, `nnodes`, `gpu`, `local`, `deepspeedconf`